tools/autograd/gen_variable_type.py

```diff
@@ -979,9 +1108,43 @@ def find_info(f: NativeFunction) -> Tuple[Optional[DifferentiabilityInfo], bool]
     result: List[NativeFunctionWithDifferentiabilityInfo] = []
     for f in native_functions:
         info, is_exact_match = find_info(f)
+
+        # Currently, the '.strides()' to 'strides_or_error' replacement does not support
+        # 'self' derivatives of an inplace function, so we must check for this case.
```

Comment:
Why doesn't it support it?

Reply:
This is not added in this PR.
This was deleted by mistake in the previous refactor and will be re-added on master independently of this PR. It will disappear form hear once rebased on top of that.